airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/test/steps/common.py

```diff
@@ -119,10 +120,6 @@ class VersionIncrementCheck(VersionCheck):
 
     @property
     def should_run(self) -> bool:
-        # Skip if connector opts out of version checks
```

Comment:
Keep this! This particular version increment check is part of the test pipeline that signals _if_ the version has been incremented. This check indeed should be skipped if `metadata.ab_internal.requireVersionIncrementsInPullRequests` is set to `False`.